opentelemetry-api/tests/test_loader.py

```diff
@@ -49,7 +49,7 @@ def setUp(self):
 
     def test_get_default(self):
         tracer = trace.tracer()
-        self.assertIs(type(tracer), trace.Tracer)
+        self.assertIs(type(tracer), trace.DefaultTracer)
```

Comment:
@Oberon00 might want to check that this is testing the right thing still. It seems like `DummyTracer` should still extend `Tracer` here, but that seems to rely on instantiating `Tracer`s.

Reply:
Alternatively here I can implement the abstract methods in `DummyTracer` and that should work as well

Reply:
I think implementing the abstract methods would be worth it here.